server/data/licenceClientTypes.ts

```diff
@@ -0,0 +1,24 @@
+import { Licence } from './licenceTypes'
+
+export interface Case {
+  licence: Licence
+  booking_id: number
```

Comment:
If we get more TS coverage we could map these names using a column alias to camelCase